frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx

```diff
@@ -437,9 +436,20 @@ export function ClustersTable(props: {
         },
         variant: 'bulk-action',
       },
-    ],
-    [modalColumns, t]
-  )
+    ]
+    if (hasAllAutomationTemplates) {
+      actions.splice(4, 0, {
+        id: 'removeAutomationTemplates',
+        title: t('Remove automation templates'),
+        click: (managedClusters: Array<Cluster>) => {
+          if (!managedClusters) return
+          setRemoveAutomationTemplates(managedClusters)
+        },
+        variant: 'bulk-action',
+      })
+    }
+    return actions
+  }, [modalColumns, hasAllAutomationTemplates, t])
```

Comment:
@jeswanke This is not what I was asking for. Currently we do not filter bulk actions. This is so that you can select all and easily apply an action to all clusters that it is applicable for.

I was suggesting that you remove the action on the table row kebab only. This is what I meant by "individual row action"